calibration_assistant.cpp

```diff
@@ -704,6 +705,8 @@ bool CalibrationAssistant::PerformSlew(double ra, double dec)
         else
         {
             m_isSlewing = false;
+            ShowStatus(_("Pausing..."));
+            wxMilliSleep(2000);

```

Comment:
Seems somewhat arbitrary? Are we waiting for a specific guide state -- if so can we wait for that state explicitly rather than waiting 2s for something?
In any case could you add a comment to the code about why this sleep is needed.

Reply:
Mount controllers/drivers can report premature slew-completions.  This is normally not a problem for anyone but it can ruin a calibration.  Since the goal of the CA is to maximize the likelihood of a successful result, a couple of additional delays seem like a small price to pay.